FUZZING.md

```diff
@@ -80,3 +80,47 @@ If both sets of tests pass, refresh the fuzzer corpora with `refresh_ssl_corpora
 cd fuzz
 ./refresh_ssl_corpora.sh /path/to/fuzzer/mode/build /path/to/non/fuzzer/mode/build
 ```
+
+## Adding New Fuzz Test Targets
+When adding new functionality, adding new fuzz tests are important to provide additional testing and verification that we are correct.
+
+### Steps
+1. `NEW_FUNCTION='name_of_new_fuzzing_target'`
+2. `touch fuzz/${NEW_FUNCTION}.cc` \
+    Write fuzzing code for libfuzzer to parse in `fuzz/${NEW_FUNCTION}.cc`. 
```

Comment:
Is the `parse(serialize and deserialize)` the only purpose of step 2? I might have added duplicate questions in `fuzz/ocsp.cc`.

Reply:
Yes, I'll clarify that in the fuzzing runbook. I initially had the same question when first creating the fuzzing test file.